src/tests/rpp/test_observables.cpp

```diff
@@ -183,3 +183,39 @@ TEST_CASE("base observables")
         }
     }
 }
+
+TEST_CASE("create observable works properly as observable")
+{
+    auto observable = rpp::source::create<int>([&](auto&& observer)
+    {
+        observer.on_next(1);
+        observer.on_completed();
+    });
+
+    SECTION("using const& variant")
```

Comment:
actually they are both `const &` due to it is observable's ref qualifier. 
`const &` would be `observable.pipe(...)`
`&&` would be `std::move(observable).pipe()` or `rpp::source::create(...).pipe()`

Reply:
oh yes indeed I confused myself with operands, not sure why code coverage didn't decrease

Reply:
Because it was totally "not created" due to it is templated function -> no instantiations -> no actual "creation"